python/Makefile

```diff
@@ -97,7 +97,7 @@ setup_linter:
 
 run_linter:
 	black \
-		./ ../testing \
+		./ ../testing ../operator/helm \
```

Comment:
This change should also be done to the pre-commit hook to ensure that the commit hook and the integration tests are aligned on what's linted

Reply:
can do - which file?

Reply:
its the .pre-commit-config.yaml in the top level directory

Reply:
will add to next commit - just finishing e2e tests